src/node.js

```diff
@@ -1,14 +1,15 @@
 export default class HeimdallNode {
   constructor(heimdall, id, data) {
     this._heimdall = heimdall;
+    this._stopped = false;
 
     this._id = heimdall.generateNextId();
     this.id = id;
 
     // TODO: strip for production builds for perfz
-    if (!(typeof this.id === 'object' && this.id !== null && typeof this.id.name === 'string')) {
-      throw new TypeError('HeimdallNode#id.name must be a string');
-     }
+    // if (!(typeof this.id === 'object' && this.id !== null && typeof this.id.name === 'string')) {
+    //  throw new TypeError('HeimdallNode#id.name must be a string');
+    // }
```

Comment:
Let's just delete this.  If we want to add back in expensive assertions we'll need a .dev, .prod build